sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -1170,6 +1182,24 @@ public void run() {
         return future;
     }
 
+    /**
+     * Implements {@link #setAuthToken} at instance level.
+     */
+    private void setInstanceAuthToken(String authToken) {
+        AuthTokenContext authTokenContext = AuthTokenContext.getInstance();
+        JwtClaims claims = null;
+        if (authToken != null) {
+            claims = JwtClaims.parse(authToken);
+        }
+        if (claims != null) {
+            AppCenterLog.debug(LOG_TAG, "Token has been refreshed.");
```

Comment:
To match the other log in the method

```suggestion
            AppCenterLog.debug(LOG_TAG, "Authentication token has been refreshed.");
```